New alien faction: the Bunrodea (#6438)
endless-sky/endless-sky: data/bunrodea/bunrodea outfits.txt

	"relay upgrades" -1
	description "The shield relay limiter is a component that can be installed into a Bunrodean shield relay in order to modify its effectiveness. Limiters increase the energy efficiency of a relay, but at the cost of reducing the strength of its shield generation."

outfit "Small Shield Relay"
	category "Systems"
	cost 343000
	thumbnail "outfit/small shield relay"
	"mass" 21
	"outfit space" -21
	"shield generation" 0.9
	"shield energy" 0.55
	"relay upgrades" 6
	description "Bunrodean shield relays are impressively effiecent; while a similarly sized shield generator from another faction might be capable of providing more shield generation, Bunrodean shield relays provide a modest amount of shield generation for only a small amount of energy."
	description "	Shield relays are able to be upgraded by installing shield boosters or limiters, which alter the performance of the shield relay. Each relay only has a limited number of slots for these upgrades, though."

outfit "Large Shield Relay"
	category "Systems"
	cost 1078000
	thumbnail "outfit/large shield relay"
	"mass" 56
	"outfit space" -56
	"shield generation" 2.75
	"shield energy" 1.4
	"relay upgrades" 16

> 	description "Some Bunrodean military captains argued that trading shield generation for energy efficiency is a foolish trade, while merchant captains looking to get the best out of their reactors argued that they could be more energy efficient. As such, shield relays are able to be customized to the owner's liking using external components."

Reviewer: ```suggestion
	description "Some Bunrodean military captains argued that trading shield generation for energy efficiency is a foolish trade, while merchant captains looking to get the most out of their reactors argued that they could be more energy efficient. As such, shield relays are able to be customized to the owner's liking using external components."
```

	description "	Shield relays are able to be upgraded by installing shield boosters or limiters, which alter the performance of the shield relay. Each relay only has a limited number of slots for these upgrades, though."

outfit "Nanite Enhancer"
	category "Systems"
	cost 76000
	thumbnail "outfit/nanite enhancer"
	"mass" 5
	"outfit space" -5
	"hull repair rate" 0.25
	"hull energy" 0.75
	"nanite upgrades" -2
	description "The nanite enhancer is a component that can be installed into a Bunrodean nanite fabricator in order to modify its effectiveness. Enhancers increase the strength of the hull repair of an fabricator, but at the cost of reduced energy efficiency."

outfit "Nanite Limiter"
	category "Systems"
	cost 21000
	thumbnail "outfit/nanite limiter"
	"mass" 2
	"outfit space" -2
	"hull repair rate" -0.02
	"hull energy" -0.17
	"nanite upgrades" -1

> 	description "The nanite limiter is a component that can be installed into a Bunrodean nanite fabricators in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Reviewer: ```suggestion
	description "The nanite limiter is a component that can be installed into a Bunrodean nanite fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."
```

outfit "Small Nanite Fabricator"
	category "Systems"
	cost 650000
	thumbnail "outfit/small nanite fabricator"
	"mass" 23
	"outfit space" -23
	"hull repair rate" 0.9
	"hull energy" 1.5
	"nanite upgrades" 4
	description "Although Bunrodea ships tend to rely on their shields for defense, some pilots will opt to install a nanite fabricator. These fabricators produce a steady stream of nanites that travel across the hull of the ship and repair any damage that they find."
	description "	Nanite fabricators are able to be upgraded by installing nanite enhancers or limiters, which alter the performance of the fabricator. Each fabricator only has a limited number of slots for these upgrades, though."

outfit "Large Nanite Fabricator"
	category "Systems"
	cost 1940000
	thumbnail "outfit/large nanite fabricator"
	"mass" 58
	"outfit space" -58
	"hull repair rate" 2.65
	"hull energy" 4.4
	"nanite upgrades" 10

> 	description "The creation of nanite fabricators was spurred on by early contact between the Bunrodea and Korath. The hull-walking robots of Korath System Cores allowed Korath ships to make repairs without the need of a spaceport, a trait that the Bunrodea favored for use in their scout vessesl."

Reviewer 2: Keeping the terminology consistent as with #6891 
```suggestion
	description "The creation of nanite fabricators was spurred on by early contact between the Bunrodea and Korath. The hull-walking microbots of Korath System Cores allowed Korath ships to make repairs without the need of a spaceport, a trait that the Bunrodea favored for use in their scout vessesl."
```

> 	description "	Nanite fabricators are able to be upgraded by installing nanite enhancers or limiters, which alter the performance of the fabricator. Each fabricator only has a limited number of slots for these upgrades, though."

Reviewer: ```suggestion
	description "	Nanite fabricators are able to be upgraded by installing nanite enhancers or limiters, which alter the performance of the fabricator. Each fabricator only has a limited number of slots for these upgrades, however."
```

	description "	Shield relays are able to be upgraded by installing shield boosters or limiters, which alter the performance of the shield relay. Each relay only has a limited number of slots for these upgrades, though."

outfit "Nanite Enhancer"
	category "Systems"
	cost 76000
	thumbnail "outfit/nanite enhancer"
	"mass" 5
	"outfit space" -5
	"hull repair rate" 0.25
	"hull energy" 0.75
	"nanite upgrades" -2
	description "The Nanite Enhancer is a component that can be installed into a Bunrodean nanite fabricator in order to modify its effectiveness. Enhancers increase the strength of the hull repair of an fabricator, but at the cost of reduced energy efficiency."

outfit "Nanite Limiter"
	category "Systems"
	cost 21000
	thumbnail "outfit/nanite limiter"
	"mass" 2
	"outfit space" -2
	"hull repair rate" -0.025
	"hull energy" -0.2
	"nanite upgrades" -1

> 	description "The Nanite Limiter is a component that can be installed into a Bunrodean Nanite Fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Reviewer 3: If there are multiple types of fabricators, it's generic. If there is only one, I'd say keep it capitalized.

> 	description "The Nanite Limiter is a component that can be installed into a Bunrodean Nanite Fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Reviewer: There are small and large types of nanite fabricators, does that count as generic?

> 	description "The Nanite Limiter is a component that can be installed into a Bunrodean Nanite Fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Reviewer 3: I guess it does.

> 	description "The Nanite Limiter is a component that can be installed into a Bunrodean Nanite Fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Reviewer: @Amazinite ?

> 	description "The Nanite Limiter is a component that can be installed into a Bunrodean Nanite Fabricator in order to modify its effectiveness. Limiters increase the energy and heat efficiency of a fabricator, but at the cost of reducing the strength of its hull repair."

Author: Looking to the Coalition module outfit names, which similarly have small and large versions, the names of the outfits aren't often used in the descriptions themselves, but where they are used (e.g. "Large Reverse Module"), they are capitalized.

#	ammo "Nanite Limiter"
	description "The creation of nanite fabricators was spurred on by early contact between the Bunrodea and Korath. The hull-walking microbots of Korath System Cores allowed Korath ships to make repairs without the need of a spaceport, a trait that the Bunrodea favored for use in their scout vessels."
	description "	Nanite fabricators are able to be upgraded by installing nanite enhancers or limiters, which alter the performance of the fabricator. Each fabricator only has a limited number of slots for these upgrades, however."

outfit "Solar Battery"
	category "Power"
	cost 49000
	thumbnail "outfit/solar battery"
	"mass" 7
	"outfit space" -7
	"energy capacity" 6000
	"solar cell" 36
#	ammo "Solar Cell"
	description "Bunrodean solar batteries are required for the installation of solar cells, but solar batteries are perfectly capable of going without solar cells. Many captains will install batteries to store energy gained from normal reactors without ever installing solar cells."

outfit "Solar Cell"
	category "Power"
	cost 17000
	thumbnail "outfit/solar cell"
	"mass" 1
	"outfit space" -1
	"solar collection" 0.145
	"solar heat" 0.07
	"solar cell" -1

> 	description "Bunrodean solar panels are extremely efficient, even being capable of powering smaller or less powerful warships. But be warned: while these solar cells can provide significant energy, that is not without the drawback of also generating waste heat that scales with the energy generated, which can be especially hazardous when near bright stars."

Reviewer: ```suggestion
	description "Bunrodean solar panels are extremely efficient, and are even capable of powering smaller or less powerful warships. But be warned: while these solar cells can provide significant energy, that is not without the drawback of also generating waste heat that scales with the energy generated, which can be especially hazardous when near bright stars."
```

outfit "Electroweak Reactor"
	category "Power"
	cost 8393000
	thumbnail "outfit/electroweak reactor"
	"mass" 106
	"outfit space" -106
	"energy generation" 17.25
	"heat generation" 20.5
	"reactor upgrades" 18
#	ammo "Reactor Overclocker"
#	ammo "Reactor Limiter"
	description "The existence of the exotic stars that this Bunrodean reactor bears the name of is still unknown to humanity. An electroweak star is an exotic star that would be prevented from collapsing by the electroweak force as quarks are converted to leptons in the star's core."
	description "	Bunrodean reactors are able to be upgraded by installing Reactor Overclockers or Limiters, which alter the performance of the reactor. Each reactor only has a limited number of slots for these upgrades, though."

outfit "Dark Reactor"
	category "Power"
	cost 15022000
	thumbnail "outfit/dark reactor"
	"mass" 142
	"outfit space" -142
	"energy generation" 24.7
	"heat generation" 28.6
	"reactor upgrades" 30
#	ammo "Reactor Overclocker"
#	ammo "Reactor Limiter"

> 	description `The term "dark star" can have different meanings, but one definition of it is a star whose core is formed around a large amount of dark matter. Such exotic stars are theorized by humanity to have possibly existed among the first generations of stars in the universe, with concentrations of dark matter acting as nucleation sites.`

Reviewer 4: I'm not sure what your personal preference is for quotation, it seems most here are ("), except here.

Author: Because the description itself contains quotation marks.

Reviewer 4: Durr.

	description "	Bunrodean reactors are able to be upgraded by installing Reactor Overclockers or Limiters, which alter the performance of the reactor. Each reactor only has a limited number of slots for these upgrades, though."

outfit "Lasher Pistol"
	category "Hand to Hand"
	cost 230000
	thumbnail "outfit/lasher pistol"
	"capture attack" 2.6
	"capture defense" 1.4

> 	description "During their wars with the Korath, it was not uncommon for Bunrodean vessels to be boarded by Korath raiders looking for technology and supplies to steal. As such, effective small arms weapons became an essential part of every Bunrodean crew to fight off any potential boarding parties."

Reviewer: ```suggestion
	description "During their wars with the Korath, it was not uncommon for Bunrodean vessels to be boarded by Korath raiders looking for technology and supplies to steal. As such, effective small arms weapons became an essential part of every Bunrodean crew that may need to fight off any boarding parties."
```

outfit "Chiisana Rift Thruster"
	category "Engines"
	cost 130000
	thumbnail "outfit/chiisana rift thruster"
	"mass" 11
	"outfit space" -11
	"engine capacity" -11
	"thrust" 6.25
	"thrusting energy" 1.15
	"thrusting heat" 0.65
	"flare sprite" "effect/bunrodea flare/large"
		"frame rate" 1
		"rewind"
		"scale" 0.3
	"flare sound" "ion tiny"
	description "While the Korath favor raw power in their engines, the Bunrodea favor efficiency, opting to make use of engines that produce less thrust for significantly less heat produced."

outfit "Nami Rift Thruster"
	category "Engines"
	cost 262000
	thumbnail "outfit/nami rift thruster"
	"mass" 19
	"outfit space" -19
	"engine capacity" -19
	"thrust" 12.1
	"thrusting energy" 2.3
	"thrusting heat" 1.3
	"flare sprite" "effect/bunrodea flare/large"
		"frame rate" 1
		"rewind"
		"scale" 0.6
	"flare sound" "ion small"

> 	description "Although Rift engines are extremely heat efficient, their energy requirement is rather high compared to the thrust they produce, even when compared to terribly inefficient Korath engines."

Reviewer: ```suggestion
	description "Although Rift engines are extremely heat efficient, their energy requirement is rather high compared to the thrust they produce, even next to terribly inefficient Korath engines."
```

outfit "Ookii Rift Thruster"
	category "Engines"
	cost 512000
	thumbnail "outfit/ookii rift thruster"
	"mass" 32
	"outfit space" -32
	"engine capacity" -32
	"thrust" 22.9
	"thrusting energy" 4.4
	"thrusting heat" 2.5
	"flare sprite" "effect/bunrodea flare/large"
		"frame rate" 1
		"rewind"
		"scale" 0.8
	"flare sound" "ion medium"

> 	description "Even though the energy drawn per unit thrust of Rift engines is poorer than that of Korath engines, the overall energy requirement of Rift engines is still far lesser by comparison."

Reviewer: ```suggestion
	description "Even though the energy drawn per unit thrust of rift engines is poorer than that of Korath engines, the overall energy requirement of rift engines is still far lesser by comparison."
```

Reviewer 3: Okay, this is an interesting one. Is `Rift` a description of the technology (ion engine), or some marketing / company name?

outfit "Subarashii Rift Thruster"
	category "Engines"
	cost 1004000
	thumbnail "outfit/subarashii rift thruster"
	"mass" 55
	"outfit space" -55
	"engine capacity" -55
	"thrust" 46.5
	"thrusting energy" 8.6
	"thrusting heat" 4.9
	"flare sprite" "effect/bunrodea flare/large"
		"frame rate" 1
		"rewind"
	"flare sound" "ion large"

> 	description "Given the lower size and thrust of rift engines, Bunrodean ships aren't well known for their pursuit capabilities. But there are rumors among some Bunrodea of an ancient cache of engines more powerful than even those of the Korath that would allow any ship to zip across the stars like a comet and chase down any potential target."

Reviewer: ```suggestion
	description "Given the lower size and thrust of rift engines, Bunrodean ships aren't well known for their pursuit capabilities. But there are rumors among some Bunrodea of an ancient cache of engines, more powerful than even those of the Korath, that would allow any ship to zip across the stars like a comet and chase down any potential target."
```

> 	description "Given the lower size and thrust of Rift engines, Bunrodean ships aren't well known for their pursuit capabilities. But there are rumors among some Bunrodea of an ancient cache of engines more powerful than even those of the Korath that would allow any ship to zip across the stars like a comet and chase down any potential target."

Reviewer: ```suggestion
	description "Given the lower size and thrust of rift engines, Bunrodean ships aren't well known for their pursuit capabilities. But there are rumors among some Bunrodea of an ancient cache of engines more powerful than even those of the Korath that would allow any ship to zip across the stars like a comet and chase down any potential target."
```

outfit "Chiisana Rift Steering"
	category "Engines"
	cost 201000
	thumbnail "outfit/chiisana rift steering"
	"mass" 17
	"outfit space" -17
	"engine capacity" -17
	"turn" 539
	"turning energy" 2.6
	"turning heat" 1

> 	description "Given that Bunrodean ships are typically focused on making heavy use of their forward facing guns, the Bunrodea favor high turn rates much more than high speeds. As such, Rift steering systems are larger than the respective thruster systems of the same size category."

Reviewer: ```suggestion
	description "Given that Bunrodean ships are typically focused on making heavy use of their forward facing guns, the Bunrodea favor high turn rates much more than high speeds. As such, rift steering systems are larger than the respective thruster systems of the same size category."
```

outfit "Nami Rift Steering"
	category "Engines"
	cost 442000
	thumbnail "outfit/nami rift steering"
	"mass" 31
	"outfit space" -31
	"engine capacity" -31
	"turn" 1137
	"turning energy" 5.3
	"turning heat" 2.1

> 	description "As with many Bunrodean systems, Rift engines produce very little heat. This allows their ships to forgo the use of any significant cooling systems, leaving more room for other systems, be they defensive or offensive."

Reviewer: ```suggestion
	description "As with many Bunrodean systems, rift engines produce very little heat. This allows their ships to forgo the use of any significant cooling systems, leaving more room for other systems, be they defensive or offensive."
```

outfit "Ookii Rift Steering"
	category "Engines"
	cost 940000
	thumbnail "outfit/ookii rift steering"
	"mass" 53
	"outfit space" -53
	"engine capacity" -53
	"turn" 2099
	"turning energy" 9.2
	"turning heat" 3.65
	description "Although the focus on steering over thrust makes Bunrodean ships easy targets to track, the power of their weapons systems would leave anyone foolish to believe that they are sitting ducks."

outfit "Subarashii Rift Steering"
	category "Engines"
	cost 1577000
	thumbnail "outfit/subarashii rift steering"
	"mass" 90
	"outfit space" -90
	"engine capacity" -90
	"turn" 3900
	"turning energy" 17.1
	"turning heat" 6.5

> 	description "The largest of the Rift steering systems, these engines are what allow the Bunrodea's heavy gunships to bear their many guns toward their unfortunate target."

Reviewer: ```suggestion
	description "The largest of the rift steering systems, these engines are what allow the Bunrodea's heavy gunships to bear their many guns toward their unfortunate target."
```
